### PR TITLE
init md highlight

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-text-editor/package.json [hunk 1]
```diff
@@ -12,5 +12,9 @@
     "vuex": "4.1.0",
     "web-client": "workspace:@ownclouders/web-client@*",
     "web-pkg": "workspace:@ownclouders/web-pkg@*"
+  },
+  "dependencies": {
+    "highlight.js": "^11.8.0",
```
Comment (pascalwengerter) [inline packages/web-app-text-editor/package.json:17]: Why are these "real" dependencies while e.g. `marked` is a peerDep? 🤔

Comment (JanAckermann) [reply packages/web-app-text-editor/package.json:17]: Just testing, this pr probably won't make it. 
But if, it must land in pkg or runtime (bc spaces also render md in description)

Comment (pascalwengerter) [reply packages/web-app-text-editor/package.json:17]: Thanks for clarification! In my opinion it'd be better to put this in a dedicated code editor extension to not blow up the default app bundle :)

Comment (JanAckermann) [reply packages/web-app-text-editor/package.json:17]: We are in decision to bundle CodiMD with the app instead. 
That is supporting syntax highlighting by default.

**Comment by JammingBen** [comment]
Since we'll be using CodiMD in the future, can we close here? Or still relevant?

**Pull request closed without merging**